learn/contributing/contributing_to_docs.md

```diff
@@ -0,0 +1,93 @@
+# Contributing to Our Documentation
+
+This documentation website is hosted in a [public GitHub repository](https://github.com/meilisearch/documentation). It is built with [VuePress](https://vuepress.github.io), written in [MarkDown](https://www.markdownguide.org/getting-started/), and deployed on [Netlify](https://www.netlify.com).
+
+## Our Documentation Philosophy
+
+Our documentation aims to be:
+
+- **Efficient**. We don't want to waste anyone's time.
+- **Accessible**. Reading this shouldn't require native English or a computer science degree.
+- **Thorough**. Behavior that isn't documented or is documented incorrectly, we consider broken.
+- **Open source**. This is a resource by MeiliSearch users, for MeiliSearch users.
+
+## How to Contribute?
+
+### Issues
+
+On the MeiliSearch documentation team, [GitHub Issues](https://github.com/meilisearch/documentation/issues/new/choose) are the primary tool we use to track tasks. They're like elements in our to-do list. Here are some good uses for issues:
+
+- A detailed bug report
+- A request for a new technical feature (e.g. dark mode, versioning, etc.)
+- A request for new content (e.g. a tutorial, guide, or explanation)
+- A helpful suggestion for the future of the documentation
+
+We love issues at MeiliSearch, because they help us do our jobs better. Nine times out of ten, the most useful contribution is a simple GitHub issue that points out a problem and proposes a solution.
+
+#### Creating Your First Issue
+
+All you need to create an issue is a [GitHub account](https://github.com).
+
+1. Log into your account.
+2. Go to the MeiliSearch [Documentation repository](https://github.com/meilisearch/documentation).
+3. Click on "Issues".
+4. Use the search bar to make sure nobody else has reported your issue already. If somebody has, give them an upvote 👍 and **don't create that issue**!
+5. Click "New issue".
+6. Fill in a short, descriptive title and longer summary of your problem or suggestion. If you're reporting a bug, make sure to include steps to reproduce the error, as well as your OS and browser version.
```

Comment:
Later you could provide templates for issues, so that when the user clicks on `new issues` he will be met with a default content the same way MeiliSearch and `meilisearch-python` have:
- https://github.com/meilisearch/meilisearch-python/pull/247/files
- https://github.com/meilisearch/MeiliSearch/tree/main/.github/ISSUE_TEMPLATE